donkeycar/templates/train.py

```diff
@@ -699,7 +699,7 @@ def multi_train(cfg, tub, model, transfer, model_type, continuous, aug):
     if model_type == "rnn" or model_type == '3d':
         train_fn = sequence_train
 
-    train_fn(cfg, tub, model, transfer, model_type, continuous, aug)
+    train_fn(cfg, tub, model, transfer, model_type, continuous)#, aug)

```

Comment:
I wouldn't remove an option in case someone wants to use it. Should't this simply pass none if arg is not provided?

Also, commenting out code is a bad practice.